mobile/src/pages/feedPage.tsx

```diff
@@ -1,14 +1,17 @@
-import { View, ScrollView } from "react-native";
+import { View, ScrollView, SafeAreaView} from "react-native";
 import BottomNavBar from "../components/BottomNavBar";
 import Card from "../components/Card";
 import TopNavBar from "../components/TopNavBar";
 
 export default function FeedPage({ navigation }) {
   return (
-    <View className="flex-1 items-center justify-center scroll">
-      <TopNavBar value="" onChangeText={() => {}} />
-      <View className="position-relative w-screen h-screen items-center justify-center top-10">
-        <ScrollView className="w-screen h-screen position-relative scroll ">
+    <View className="flex-1 h-screen items-center justify-center scroll">
+      <SafeAreaView className="flex-1">
+        <View className="grow">
+          <TopNavBar className="m-2" value="" onChangeText={() => {}} />
```

Comment:
Ah - this here won't work. You'll need to use `useState()` for this navbar search to actually work. Not a prereq for merging this though, just something to know